lib/msf/core/session/interactive.rb

```diff
@@ -142,6 +142,9 @@ def _usr1
   def _suspend
     # Ask the user if they would like to background the session
     intent = prompt_yesno("Background session #{name}?")
+    if intent
+    Readline::HISTORY.length.times {Readline::HISTORY.pop}
+    end
     if !intent
```

Comment:
Pretty minor, but this looks like it's missing an indentation here. It also looks like the conditional block can be combined with what's right after it.

```suggestion
    if intent
      Readline::HISTORY.length.times { Readline::HISTORY.pop }
    else
```

Reply:
Okay I'll fix the indentation and combine it with the next block.